app/views/home/_header.html.erb

```diff
@@ -28,6 +28,7 @@
             </a>
             <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
               <li><%= link_to 'Rewards', rewards_path, class:"dropdown-item" %></li>
+              <li><%= link_to 'My Rewards', orders_path, class:"dropdown-item" %></li>
```

Comment:
Can name them "Orders"?

Reply:
Hi Robert! Nice to e-meet you too :D. Thanks for review! I was struggling with my thoughts how to name this link. I know, this link provides actually to orders history, but  acceptance criterias in hub only say about 'list of bought rewards', so I decided to choose this name.